Petko/Zad2.cpp

```diff
@@ -0,0 +1,76 @@
+// ConsoleApplication1.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+struct OrderedPair {
+	int first;
+	int second;
+};
+
+void inputPair(OrderedPair& p)
+{
+	cin >> p.first >> p.second;
+}
+bool compairPairs(const OrderedPair& p, const OrderedPair& p1)

```

Comment:
This one has a rather confusing name. Something like `isBigger` is a lot better. Comparison allows for 3 results `smaller`, `equal`, `bigger`.. `bool` however allows to only return two values. The typical way to write a comparator is to return  a `negative number` if `p < p1`, `zero` if `p == p1` and `positive number` if `p > p1`.